provider/oci/environ.go

```diff
@@ -226,6 +228,8 @@ func (e *Environ) getOciInstances(ids ...instance.Id) ([]*ociInstance, error) {
 
 	instances, err := e.Compute.ListInstances(context.Background(), request)
 	if err != nil {
+		// todo(tsm) add ctx as arg
```

Comment:
Why not add the missing ctx arg :-)
It's needed.

Reply:
Was slightly scared of changing method signatures - wanted to confirm that's what you wanted to do :)